core/block_validator_test.go

```diff
@@ -35,7 +35,7 @@ func testChainConfig() *ChainConfig {
 		Forks: []*Fork{
 			&Fork{
 				Name:  "Homestead",
-				Block: big.NewInt(1150000),
+				Block: big.NewInt(0),
```

Comment:
btw, this `big.NewInt(0)` in all of the tests is set because all of them are expecting Homestead, but for testing purposes there're only 1-4 blocks in test blockchain.